src/test/groovy/com/palantir/gradle/versions/VersionsLockPluginIntegrationSpec.groovy

```diff
@@ -869,7 +902,13 @@ class VersionsLockPluginIntegrationSpec extends IntegrationSpec {
             # Run ./gradlew --write-locks to regenerate this file
```

Comment:
Once this progresses to the point of rolling out more broadly, I'd like to explore adding an explanation for the three blank lines here in the `versions.locks` file header

Reply:
updated header to `# Run ./gradlew --write-locks to regenerate this file. Blank lines are to minimize merge conflicts.`